### ISSUE TITLE
downloadTaifex.R

### ISSUE THREAD

**Issue by mansunkuo** [datasci-info/BeATradeR#23]
line 98:

df = dbGetQuery(conn,sprintf("select time, price, volume from FutureTByT where pcode = 'TX' and exMW = '201511'", dataDate))

> Error in sqliteSendQuery(con, statement, bind.data) : 
>   error in statement: no such table: FutureTByT

**Comment by wush978** [comment]
我認為原因是 #22

**Comment by mansunkuo** [comment]
看一下好像是喔!!!

**Comment by lovelybigdata** [comment]
是的 檔案下載不成功，TaifexDaily.sqlite 只佔用了 0kb 的空間，所以對它做任何的 sql 查詢都會報錯，同樣的問題也會出現在  Example1.R 中
